src/main/java/wordcloud/nlp/FrequencyAnalizer.java

```diff
@@ -95,7 +99,39 @@
         return sorted.subList(0, Math.min(sorted.size(), wordFrequencesToReturn));
     }
 
-    public void setStopWords(Collection<String> stopWords) {
+    public List<WordFrequency> load(final List<String> texts,FileType fileType) {
+    	if(fileType == FileType.REGULAR)
```

Comment:
I like to always wrap conditional statements in brackets. 
```java
if(fileType == FileType.REGULAR) {
    return load(texts);
}
```
for short-simple statements like this I also approve of:
```java
if(fileType == FileType.REGULAR) { return load(texts); }
```